internal/iam/command_ipgroup.go

```diff
@@ -5,16 +5,17 @@ import (
 	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
 	"github.com/confluentinc/cli/v3/pkg/output"
 	"github.com/spf13/cobra"
+	"strings"
 )
 
 type ipGroupCommand struct {
 	*pcmd.AuthenticatedCLICommand
 }
 
 type ipGroupOut struct {
-	ID         string   `human:"ID" serialized:"id"`
-	GroupName  string   `human:"Group Name" serialized:"group_name"`
-	CidrBlocks []string `human:"CIDR Blocks" serialized:"cidr_blocks"`
+	ID         string `human:"ID" serialized:"id"`
+	GroupName  string `human:"Name" serialized:"group_name"`
```

Comment:
The command is called `ip-group`, so it's clear that we're listing groups.
```suggestion
	Name       string `human:"Name" serialized:"name"`
```